### PR TITLE
JDK-8290887 Unused private method in TrustManagerFactoryImpl

### PR DESCRIPTION
**Pull request by mcpowers**
https://bugs.openjdk.org/browse/JDK-8290887

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8290887](https://bugs.openjdk.org/browse/JDK-8290887): Unused private method in TrustManagerFactoryImpl

### Reviewers
 * [Weijun Wang](https://openjdk.org/census#weijun) (@wangweij - **Reviewer**) ⚠️ Review applies to [6d466322](https://git.openjdk.org/jdk/pull/9616/files/6d466322db0d9a8002a131b365e099a1ea5aa2fb)
 * [Xue-Lei Andrew Fan](https://openjdk.org/census#xuelei) (@XueleiFan - **Reviewer**) ⚠️ Review applies to [6d466322](https://git.openjdk.org/jdk/pull/9616/files/6d466322db0d9a8002a131b365e099a1ea5aa2fb)
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9616/head:pull/9616` \
`$ git checkout pull/9616`

Update a local copy of the PR: \
`$ git checkout pull/9616` \
`$ git pull https://git.openjdk.org/jdk pull/9616/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9616`

View PR using the GUI difftool: \
`$ git pr show -t 9616`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9616.diff">https://git.openjdk.org/jdk/pull/9616.diff</a>

</details>

### CODE REVIEW

**Comment by mcpowers** [comment]
Thanks.

**Review by XueleiFan — APPROVED** [review]

File: src/java.base/share/classes/sun/security/ssl/TrustManagerFactoryImpl.java [hunk 1]
```diff
@@ -105,30 +105,6 @@ protected TrustManager[] engineGetTrustManagers() {
         return new TrustManager[] { trustManager };
     }
 
-    /*
```
Comment (seanjmullan) [inline src/java.base/share/classes/sun/security/ssl/TrustManagerFactoryImpl.java:108]: I think you can also remove the `import java.io.*` on line 28 now that this block of code has been removed.

**Review by seanjmullan — APPROVED** [review]

**Pull request closed without merging**